components/speedreader/features.cc

```diff
@@ -11,12 +11,15 @@
 namespace speedreader {
 
 const base::Feature kSpeedreaderFeature {
-      "Speedreader",
+  "Speedreader",
```

Comment:
looks like bad spacing, needs clang-format ?

Reply:
clang-format also did this